forge-game/src/main/java/forge/game/staticability/StaticAbility.java

```diff
@@ -583,6 +583,13 @@ public int compareTo(StaticAbility arg0) {
         .result();
     }
 
+    public long getTimestamp() {
+        if (hasParam("Timestamp")) {
+            return Long.valueOf(getParam("Timestamp"));
```

Comment:
technically you already handled "command zone pollution" with `setRenderForUI`
but it might still be better for performance if we avoid extra Card objects anyway 🤔

so without effect as hostCard it will need to use the affected Card I guess?
need to think about any problems that might cause...

Reply:
Yes, this part is only for stuff that affects the card itself (counters, suspected, maybe unearth)

it's also for preventing problems on LKI copies, where i would have needed to copy the effects too